contracts/eth/nearprover/utils/upgrade_bridge_address.js

```diff
@@ -0,0 +1,53 @@
+require('dotenv').config();
+
+const { ethers } = require('hardhat');
+const { assert, expect } = require('chai');
+
+const BRIDGE_ADDRESS_SLOT = 2;
+
+async function upgradeProversBridgeAddressTo(provider, proverAddress, newBridgeAddress) {
+    const nearProverFactory = await ethers.getContractFactory('NearProver');
+    const nearProver = nearProverFactory.attach(proverAddress);
+
+    console.log(`Got prover at address: ${proverAddress}`);
+
+    const initialBridgeAddress = await nearProver.bridge();
+    console.log(`Initial bridge address: ${initialBridgeAddress}`);
+    console.log(`Trying to upgrade bridge address to: ${newBridgeAddress}`);
+
+    const adminWallet = new ethers.Wallet(process.env.ETH_PRIVATE_KEY, provider);
```

Comment:
We should consider allowing hardware wallet, or offline signing in this step, to avoid compromising the key during the upgrade process.

Reply:
Sure, this is already WIP and will be added really soon.